Error creating cooked build with ContainsNaN function

1) Launch New Project in UE4.19
2) Enable FaceFX plugin
3) Click "Launch" from the Editor

Error message below indicates that the problem relates to this [commit](https://github.com/FaceFX/FaceFX-UE4/commit/de95aa4d217c2140eaadbd0e7b9d890cdd412e67) that created the ContainsNaN function.

```
LogPlayLevel: ********** BUILD COMMAND STARTED **********
LogPlayLevel: Completed Launch On Stage: Build Task, Time: 1.263398
LogPlayLevel: Running: C:\Program Files\Epic Games\UE_4.19\Engine\Binaries\DotNET\UnrealBuildTool.exe Slade419 Win64 Development -Project="C:\Users\Doug\Documents\Unreal Projects\Slade419\Slade419.uproject"  "C:\Users\Doug\Documents\Unreal Projects\Slade419\Slade419.uproject" -NoUBTMakefiles  -remoteini="C:\Users\Doug\Documents\Unreal Projects\Slade419" -ski
pdeploy -ini:Game:[/Script/UnrealEd.ProjectPackagingSettings]:BlueprintNativizationMethod=Disabled -noxge -generatemanifest -NoHotReload
LogPlayLevel: Took 1.5426072s to run UnrealBuildTool.exe, ExitCode=0
LogPlayLevel: Running: C:\Program Files\Epic Games\UE_4.19\Engine\Binaries\DotNET\UnrealBuildTool.exe Slade419 Win64 Development -Project="C:\Users\Doug\Documents\Unreal Projects\Slade419\Slade419.uproject"  "C:\Users\Doug\Documents\Unreal Projects\Slade419\Slade419.uproject" -NoUBTMakefiles  -remoteini="C:\Users\Doug\Documents\Unreal Projects\Slade419" -ski
pdeploy -ini:Game:[/Script/UnrealEd.ProjectPackagingSettings]:BlueprintNativizationMethod=Disabled -noxge -NoHotReload -ignorejunk
LogPlayLevel:   Performing 1 actions (4 in parallel)
LogPlayLevel:   [1/1] Link Slade419.exe
**LogPlayLevel:   UE4-FaceFX.lib(Module.FaceFX.cpp.obj) : error LNK2005: "bool __cdecl ContainsNaN(class TArray<struct FBoneTransform,class FDefaultAllocator> const &)" (?ContainsNaN@@YA_NAEBV?$TArray@UFBoneTransform@@VFDefaultAllocator@@@@@Z) already defined in UE4-AnimGraphRuntime.lib(Module.AnimGraphRuntime.cpp.obj)**
LogPlayLevel:      Creating library C:\Users\Doug\Documents\Unreal Projects\Slade419\Binaries\Win64\Slade419.lib and object C:\Users\Doug\Documents\Unreal Projects\Slade419\Binaries\Win64\Slade419.exp
LogPlayLevel:   C:\Users\Doug\Documents\Unreal Projects\Slade419\Binaries\Win64\Slade419.exe : fatal error LNK1169: one or more multiply defined symbols found
LogPlayLevel:   ERROR: UBT ERROR: Failed to produce item: C:\Users\Doug\Documents\Unreal Projects\Slade419\Binaries\Win64\Slade419.exe
```

--- a/Source/FaceFX/Private/Animation/AnimNode_BlendFaceFXAnimation.cpp
+++ b/Source/FaceFX/Private/Animation/AnimNode_BlendFaceFXAnimation.cpp
@@ -176,19 +176,6 @@ void FAnimNode_BlendFaceFXAnimation::Evaluate_AnyThread(FPoseContext& Output)
 #endif
 }
 
-bool ContainsNaN(const TArray<FBoneTransform> & BoneTransforms)
-{
-	for (int32 i = 0; i < BoneTransforms.Num(); ++i)
-	{
-		if (BoneTransforms[i].Transform.ContainsNaN())
-		{
-			return true;
-		}
-	}
-
-	return false;
-}
-
 void FAnimNode_BlendFaceFXAnimation::EvaluateComponentSpace_AnyThread(FComponentSpacePoseContext& Output)
 {
 	SCOPE_CYCLE_COUNTER(STAT_FaceFXBlend);

